app/views/properties/_property_details_top_section.html.erb

```diff
@@ -45,6 +45,12 @@
           </p>
           <% end %>
 
+          <% if property.is_tmo? %>
+          <p class="govuk-body-s hackney-property-tenure hackney-property-tenure-orange">
```

Comment:
Well now we're abusing the `hackney-property-tenure` classes to do TMO. Can you add some `FIXME:` at least?